released/packages/coq-verified-extraction/coq-verified-extraction.0.9/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "Matthieu Sozeau <matthieu.sozeau@inria.fr"
+homepage: "https://github.com/yforster/coq-verified-extraction"
+dev-repo: "git+https://github.com/yforster/coq-verified-extraction"
+bug-reports: "https://github.com/yforster/coq-verified-extraction/issues"
+authors:  "Yannick Forster"
+          "Matthieu Sozeau"
+          "Nicolas Tabareau"
+  ]
+license: "MIT"
+build: [
+  [make]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml"
+  "malfunction"
```

Comment:
Since you are waiting for a specific version of `malfunction`, it should appear in the dependency constraints.

Reply:
Yep, I don't know which one yet, hence the draft

Reply:
Oh, okay. I thought it was draft just because you were waiting for Malfunction 0.5 to be merged into the main Opam repository.